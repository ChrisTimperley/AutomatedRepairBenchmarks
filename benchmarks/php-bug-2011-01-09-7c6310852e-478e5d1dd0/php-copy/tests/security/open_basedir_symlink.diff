034+ 
035+ Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 21
036+ bool(false)
037+ 
038+ Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/symlink.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 22
039+ bool(false)
035- bool(true)
036- bool(true)
037- bool(true)
038- bool(true)
041+ 
042+ Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 26
043+ bool(false)
044+ 
045+ Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/ok2/ok.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 27
046+ bool(false)