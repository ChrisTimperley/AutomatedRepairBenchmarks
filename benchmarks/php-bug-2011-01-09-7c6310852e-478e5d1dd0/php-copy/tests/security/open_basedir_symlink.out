*** Testing open_basedir configuration [symlink] ***
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad/symlink.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 7
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/symlink.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 8
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad/symlink.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 9
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/symlink.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 10
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad/bad.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 13
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 14
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad/bad.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 15
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad/bad.txt) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 16
bool(false)

Warning: symlink(): open_basedir restriction in effect. File(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/bad) is not within the allowed path(s): (.) in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 17
bool(false)

Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 21
bool(false)

Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/symlink.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 22
bool(false)
bool(true)

Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 26
bool(false)

Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/ok2/ok.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_symlink.php on line 27
bool(false)
*** Finished testing open_basedir configuration [symlink] ***